packages/database/src/migrations/20231031011521-RemoveNullableFromProjectConfig-modifies-schema.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`ALTER TABLE project ALTER COLUMN config SET NOT NULL`);
```

Comment:
Our types package plays nicely when we set fields to be not null, and in this case we never really expect the config to be `NULL` ever (empty object represents empty state)